edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -28,13 +28,15 @@ class JwtTokenVersion:
     added_version = '1.1.0'
 
 
-def jwt_decode_handler(token):
+def jwt_decode_handler(token, decode_symmetric_token=True):
     """
     Decodes (and verifies) a JSON Web Token (JWT).
 
     Notes:
         * Requires "exp" and "iat" claims to be present in the token's payload.
         * Aids debugging by logging InvalidTokenError log entries when decoding fails.
+        * Setting for JWT_DECODE_HANDLER expects a single argument, token. The new argument i.e. decode_symmetric_token
```

Comment:
Minor changes so the comment remains true even when this argument is no longer _new_.
```suggestion
        * Setting for JWT_DECODE_HANDLER expects a single argument, token. The argument decode_symmetric_token
```